tox.ini

```diff
@@ -48,8 +48,8 @@ commands =
   coverage3 run --source circuit_knitting_toolbox --parallel-mode -m pytest test/ {posargs}
   coverage3 combine
   coverage3 html
-  coverage3 report --fail-under=100 --show-missing --omit="circuit_knitting_toolbox/circuit_cutting/wire_cutting/**/*,circuit_knitting_toolbox/entanglement_forging/**/*,circuit_knitting_toolbox/utils/conversion.py,circuit_knitting_toolbox/utils/metrics.py,circuit_knitting_toolbox/utils/orbital_reduction.py"
-  coverage3 report --fail-under=80
+  coverage3 report --fail-under=100 --show-missing --omit="circuit_knitting_toolbox/circuit_cutting/wire_cutting/**/*,circuit_knitting_toolbox/circuit_cutting/cutqc/**/*,circuit_knitting_toolbox/entanglement_forging/**/*,circuit_knitting_toolbox/utils/conversion.py,circuit_knitting_toolbox/utils/metrics.py,circuit_knitting_toolbox/utils/orbital_reduction.py"
+  coverage3 report --fail-under=80 --omit="circuit_knitting_toolbox/circuit_cutting/wire_cutting/**/*"
```

Comment:
I think we should count cutqc in our coverage percentage unless/until we formally deprecate it.

Reply:
cutqc would be tested under the 80% tests here, right? We just dont want to test the wire_cutting package because it's deprecated

Reply:
OK I see. I moved the cutqc tests into a test/cutting/cutqc dir, but they got lost in a local commit I guess. Will re-add tomorrow